src/containers/main/Dashboard/components/BlockExplorerMini/MinerCount/MinerCount.tsx

```diff
@@ -12,18 +12,22 @@ export default function MinerCount() {
     const { data: totalMiners } = useMinerStats();
     const contentRef = useRef<HTMLDivElement>(null);
     const contentWidth = contentRef.current?.clientWidth;
+
+    const value = totalMiners || 0;
+    const rounded = value >= 50_000 ? Math.floor(value / 1000) * 1000 : value;
+    const notation = value >= 50_000 ? 'compact' : 'standard';
```

Comment:
Should this be added to the helpers as a function or will this format only be needed in this one place?